publications/tasks.py

```diff
@@ -1,27 +1,73 @@
 from django_q.models import Schedule
-from datetime import date, timezone
 from publications.models import Publication
-from django.core.mail import send_mail
+from views import harvest_data
+import requests
+from bs4 import BeautifulSoup
+import json
+import xml.dom.minidom
+from xml.dom.minidom import parseString
+from django.db.models import Max
+from django.contrib.gis.geos import GEOSGeometry
+
+def harvest_demo():
```

Comment:
This needs to go into a test case. You can just skip it by default but still run it manually.

Reply:
Need bit of direction in this. Will discuss in today's call.